tests/foreman/cli/test_leapp_client.py

```diff
@@ -229,12 +32,16 @@ def precondition_check_upgrade_and_install_leapp_tool(custom_leapp_host):
     ids=lambda upgrade_path: f'{upgrade_path["source_version"]}'
     f'_to_{upgrade_path["target_version"]}',
 )
+@pytest.mark.parametrize('auth_type', ['admin', 'non-admin'])
 def test_leapp_upgrade_rhel(
     module_target_sat,
     custom_leapp_host,
     upgrade_path,
     verify_target_repo_on_satellite,
     precondition_check_upgrade_and_install_leapp_tool,
+    auth_type,
+    module_sca_manifest_org,
+    default_location,
```

Comment:
Instead of default_location, I'd suggest using module_location fixture

Reply:
I would like to go with default_location. The satellite for leapp would not impact any other so would like to keep it.

Reply:
I understand but using default_* entities could impact other tests, and using module_location should work the same

Reply:
I agree. Unless there's a good reason, function/module should be preferred. the only cost paid is in some amount of setup time.

Reply:
module_target_sat used here will be only for this module, so if we are going to use default location then if would not impact any other module here and this is a positive scenario which is not making any changes which will impact other tests execution. If in future we add more tests to the module which could impact other tests execution, I will use different location in other test. Please correct me if my understanding is incorrect here.